Cargo.toml

```diff
@@ -1,66 +1,37 @@
 [package]
-name = "solo-usbip"
+name = "trussed-usbip"
 version = "0.0.1"
 authors = ["Nicolas Stalder <n@stalder.io>", "Conor Patrick <conor@solokeys.com>", "Szczepan Zalega <szczepan@nitrokey.com>"]
 edition = "2021"
 
-[[bin]]
-name = "usbip-simulation"
-path = "src/bin/main.rs"
-
 [dependencies]
-chacha20 = { version = "0.7", features = ["rng"] }
-generic-array = "0.14.3"
-heapless = "0.6"
 interchange = "0.2.0"
-nb = "1"
 log = { version = "0.4.14", default-features = false }
-clap = { version = "3.0.0", features = ["derive"] }
-clap-num = "1.0.0"
-
-apdu-dispatch = "0.1"
-ctap-types = "0.1"
-ctaphid-dispatch = "0.1"
-admin-app = "0.1"
-fido-authenticator = { version = "0.1", features = ["dispatch"] }
-trussed = { version = "0.1", features = ["clients-3", "virt"] }
-
-# components
-usbd-ctaphid = { git = "https://github.com/Nitrokey/nitrokey-3-firmware" }
-
-# storage
-littlefs2 = "0.3.1"
-
-# USB related
-usbip-device = "*"
+trussed = { version = "0.1", features = ["virt"] }
 usb-device = { version = "0.2.7", default-features = false }
+usbip-device = "0.1.4"
 
-# Optional
-pretty_env_logger = { version = "0.4.0", optional = true }
+# ctaphid
+ctaphid-dispatch = { version = "0.1", optional = true }
+usbd-ctaphid = { git = "https://github.com/Nitrokey/nitrokey-3-firmware", optional = true }
 
-[features]
-default = ["no-buttons"]
-enable-logs = ["pretty_env_logger", "admin-app/log-all", "fido-authenticator/log-all"]
-
-# Use to auto-succeed every user presence check
-no-buttons = []
-
-# Reconfigure the NFC chip in any case
-reconfigure-nfc = []
+[dev-dependencies]
+clap = { version = "3.0.0", features = ["derive"] }
+clap-num = "1.0.0"
+pretty_env_logger = "0.4.0"
+trussed = { version = "0.1", features = ["clients-3"] }
 
-log-all = []
-log-none = []
-log-info = []
-log-debug = []
-log-warn = []
-log-error = []
+# applications
+admin-app = { version = "0.1", features = ["log-all"] }
+fido-authenticator = { version = "0.1", features = ["dispatch", "log-all"] }
 
+[features]
+default = ["ctaphid"]
+ctaphid = ["ctaphid-dispatch", "usbd-ctaphid"]
 
-#[profile.release]
-#codegen-units = 1
-#opt-level = "z"
-#incremental = false
-#debug = true
+[[example]]
+name = "fido"
+required-features = ["ctaphid"]
 
 [patch.crates-io]
-trussed = { git = "https://github.com/trussed-dev/trussed" }
+trussed = { git = "https://github.com/robin-nitrokey/trussed", branch = "virt-store-provider" }
```

Comment:
Since this is a side branch anyway, perhaps even better would be to keep it trussed-dev/trussed space instead. Let's discuss that on the next occassion.

Reply:
I would just wait until it is merged as this PR is not really urgent.